PSReadLine/PSReadLineResources.resx

```diff
@@ -858,4 +858,13 @@ Or not saving history with:
   <data name="FailedToConvertPointToRenderDataOffset" xml:space="preserve">
     <value>Cannot locate the offset in the rendered text that was pointed by the original cursor. Initial Coord: ({0}, {1}) Buffer: ({2}, {3}) Cursor: ({4}, {5})</value>
   </data>
+  <data name="CapitalizeWordDescription" xml:space="preserve">
+    <value>Find the next word starting from the current position and then make it Pascal case.</value>
+  </data>
```

Comment:
This shouldn't described as "Pascal case". It only changes the first character of the word. It doesn't change a token like "outputfile" to "OutputFile".

Reply:
You are right. It changes `aBC` to `Abc`, so how should we call it?

Reply:
I think it's called "title case."

Reply:
It is actually "sentence case" not "title case". But I don't think most people know the difference. To make it clear to people we should say that it "upcases the first character and downcases the remaing characters".

Reply:
Sounds good. I will submit a PR to correct the wording.